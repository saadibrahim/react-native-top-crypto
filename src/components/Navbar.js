import React, { Component } from 'react';
import { View, Text } from 'react-native';
import PropTypes from 'prop-types';
import styles from './styles/NavbarStyles';

export default class Navbar extends Component {
  render() {
    const { text } = this.props;

    return (
      <View style={styles.navbar}>
        <Text style={styles.text}>{text}</Text>
      </View>
    );
  }
}

Navbar.propTypes = {
  text: PropTypes.string.isRequired,
};
