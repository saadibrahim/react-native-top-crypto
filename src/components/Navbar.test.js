import React from 'react';
import renderer from 'react-test-renderer';

import Navbar from './Navbar';

test('renders the Navbar using Snapshots', () => {
  const component = renderer.create(<Navbar text="Test" />);
  expect(component).toMatchSnapshot();
});
