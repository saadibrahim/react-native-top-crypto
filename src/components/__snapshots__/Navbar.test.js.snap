// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Navbar using Snapshots 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2CBCF2",
      "flexDirection": "row",
      "height": 74,
      "justifyContent": "center",
      "marginTop": -44,
      "paddingTop": 30,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 16,
        "fontWeight": "600",
      }
    }
  >
    Test
  </Text>
</View>
`;
